clusterloader2/testing/experimental/storage/pod-startup/suite.yaml

```diff
@@ -0,0 +1,47 @@
+- identifier: emptydir-vol-per-pod
+  configPath: "testing/experimental/storage/pod-startup/config.yaml"
+  overridePaths:
+    - "./testing/experimental/storage/pod-startup/volume-types/emptydir/override.yaml"
```

Comment:
Does it need to be in quotation marks? Here and in all the file?

Reply:
Fixed